x-pack/plugins/security_solution/public/detections/components/rules/mitre/helpers.ts

```diff
@@ -4,15 +4,78 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { isEmpty } from 'lodash/fp';
+import { FieldHook } from 'src/plugins/es_ui_shared/static/forms/hook_form_lib';
+import { subtechniquesOptions } from '../../../mitre/mitre_tactics_techniques';
 
-import { IMitreAttack } from '../../../pages/detection_engine/rules/types';
+import { IMitreAttackTechnique } from '../../../pages/detection_engine/rules/types';
 
 export const isMitreAttackInvalid = (
   tacticName: string | null | undefined,
-  technique: IMitreAttack[] | null | undefined
+  technique: IMitreAttackTechnique[] | null | undefined
 ) => {
-  if (isEmpty(tacticName) || (tacticName !== 'none' && isEmpty(technique))) {
+  if (
+    tacticName !== 'none' &&
+    technique != null &&
+    (isEmpty(technique) || !containsTechniques(technique))
+  ) {
     return true;
   }
   return false;
 };
+
+export const isMitreTechniqueInvalid = (
```

Comment:
technically, MITRE is the name of the company, and ATT&CK is the name of the framework.
so you could say ATT&CK technique or MITRE ATT&CK technique, but MITRE technique doesn't really make sense.

i know it's not user facing code, but consistency usually helps

```suggestion
export const isMitreAttackTechniqueInvalid = (
```